DMTR-53.tex

```diff
@@ -147,47 +147,297 @@ \subsection{Summary Table}
+        self.assertIsInstance(meas, Measurement)
+        self.assertEqual(meas.metric_name, lsst.verify.Name(metric='ip_isr.IsrTime'))
+>       self.assertGreater(meas.quantity, 0.0 * u.second)
+E       AssertionError: <Quantity 0.0 s> not greater than <Quantity 0.0 s>
+
+tests/test_profiling.py:54: AssertionError
+============================= 29 tests deselected ==============================\end{verbatim}
+
+This is a unit test limitation already reported 
+in \texttt{DM-12848} and does not affect the functionality of the package.
+
 \subsection{AG-00-05}
+
+\textit{Execution of the Alert Generation Pipelines produces the expected data products: PARTIALLY PASSED}
+
+We performed a Butler \texttt{get} for \texttt{calexp}, \texttt{deepDiff\_differenceExp}, and \texttt{deepDiff\_diaSrc} dataset types using 
+dataIds constructed all visits and ccds listed in the appendix of \citeds{LDM-533}.
+
+We found several dataIds where products were not generated:  
+
+\begin{itemize}
+	\item For visits 411321, 411371, and 411422, ccds 48--60 and 62 were missing all three of \texttt{calexp}, \texttt{deepDiff\_differenceExp}, and \texttt{deepDiff\_diaSrc} products.  This failure was due to the time limit being reached on the slurm execution script (which looped sequentially over ccds for each visit).
+	\item For visit 411657, ccd 47 only was missing all three products.  Inspecting the log revealed an error:
+		\begin{verbatim}  
+File "/software/lsstsw/stack3_20171023/stack/miniconda3-4.3.21-10a4fa6/
+Linux64/meas_algorithms/14.0-7-g23fdbe95+15/python/lsst/meas/algorithms/
+measureApCorr.py", line 251, in run
+    (name, len(subset2), self.config.minDegreesOfFreedom+1))
+RuntimeError: Unable to measure aperture correction for required algorithm 
+'base_PsfFlux': only 0 sources, but require at least 2.
+\end{verbatim}
+\end{itemize}
+
+We inspected the \texttt{DIAObject} table by opening the 
+\texttt{l1db/association.db} sqlite database produced by \texttt{ap\_pipe} 
+with \texttt{sqlite3}.  Issuing the query \texttt{select * from dia\_objects limit 10;} confirmed that the database was non-empty.
+
+
 \subsection{AG-00-10}
+
+\textit{Generated PVIs include science, mask, and variance images; background model; zeropoint; PSF, and WCS: PASSED}
+
+For all of the generated PVIs, we used Butler calls to check for the existence
+image, mask, and variance planes; zeropoints, PSFs, and WCS; and background models.
+
+All dataIds where data was generated in AG-00-05 had all of the required components.
+
+\textit{Masks and background are appropriately applied: PARTIALLY PASSED}
+
+We chose two visits and five ccds at random:
+visit 411673, ccds 15, 32, 45, 55, and 57, and
+visit 410915, ccds 3,  8, 23, 41, and 44.
+
+We used the Butler and DS9 to examine these PVIs.  We noted a few unmasked 
+bad columns as well as the effect of crosstalk; crosstalk correction for DECam
+was not enabled at the time of this processing.
```

Comment:
Might be worth adding a note as to why crosstalk correction wasn't enabled.

Do we know why those bad columns weren't caught? If this is a problem we need to investigate, probably worth a JIRA ticket.